packages/orchestration/src/orchestration-stream-response.ts

```diff
@@ -41,6 +50,38 @@ export class OrchestrationStreamResponse<T> {
     this._finishReasons = finishReasons;
   }
 
+  public getToolCalls(choiceIndex = 0): MessageToolCalls | undefined {
+    try {
+      const toolCallsAccumulators =
+        this._toolCallsAccumulators.get(choiceIndex);
+      if (!toolCallsAccumulators) {
+        throw new Error(`No tool calls found for choice index ${choiceIndex}`);
```

Comment:
[q] What about return undefined? When do we return undefined?

Reply:
Hmm fair, I suppose returning undefined is the better response.